Assets/Scripts/Presenters/InputPresenter.cs

```diff
@@ -89,9 +89,26 @@ public void DiscardDeal()
             ToggleTalkButton(true);
             ToggleNewOfferButton(true);
         }
+
+        private bool CheckAmountOfferd()
+        {
+            try
+            {
+                Convert.ToInt32(offeringResourceAmount.text);
+                Convert.ToInt32(requestingResourceAmount.text);
+            }
+            catch (Exception e)
+            {
+                ShowError("you can not offer or request 0 resources");
```

Comment:
Does this catch if you put the input value as '0'? That should be an integer right?